day2.md

```diff
@@ -0,0 +1,171 @@
+
+
+
+<img src="day2/IMG_20160906_135506.jpg" width="500"><br/>
+These colorful rice intrigued me a lot. So I got them along with tofu and pork bellies!
+
+<img src="day2/14369984_1115753411807685_8891262251835998179_n.jpg" width="500"><br/>
+Matt got these amazing sushi which was too fishy to him, but it tasted like heaven to me! Definitely one of the best sushi I ever had, even including the other two sushi places we went to later in the trip. It had the perfect thickness and amount of the fish. Its texture screamed out how fresh it was and that little bit of ginger in the end of the bite ended the whole experience by taking it to another level. 🐟🐟🐟🙈🙈🙈
+
+After having fun, food, and rest, we were on our way to <a name="temple">Sensō-ji</a>: an ancient Buddhist temple.
+
+<img src="day2/IMG_20160906_143753.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_145103.jpg" width="500"><br/>
+We saw two women in Kimono and we just couldn't resist taking a picture of them. 🙈
+
+Here comes Sensō-ji. <br />
+<img src="day2/IMG_20160906_154940.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151107.jpg" width="500"><br/>
+Sensō-ji was huge: it required over an hour to just walk around the whole place. <br />
+Katherine and Matt rested in shadows while I was walking around. <br />
+<img src="day2/IMG_20160906_153415.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150819.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150810.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150831.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150924.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150959.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151518.jpg" width="500"><br/>
+Hello, Mr. Ninja!<br/>
+<img src="day2/IMG_20160906_151435.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151552.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151620.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150731.jpg" width="500"><br/>
+A lot of shops were closed when we got there. <br/>
+<img src="day2/IMG_20160906_151812.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151831.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_154606.jpg" width="500"><br/>
+<img src="day2/14379679_1115754091807617_7583104269522736177_o.jpg" width="500"><br/>
+
+It's time for us to go to <a name="shinjuku">Shinjuku</a> for dinner!<br />
+<img src="day2/14352325_1115752741807752_8991592585555306596_o.jpg" width="500"><br/>
+A women in pretty Kimono in front of a flower shop in the station.
+
+<img src="day2/IMG_20160906_171827.jpg" width="500"><br/>
+<img src="day2/14361400_1115754335140926_2182748393200978861_o.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_172802.jpg" width="500"><br/>
+We first went to Nakajima(中嶋), a [one star michelin](http://gm.gnavi.co.jp/shop/0120162810/) restaurant, without reservation to see if we were lucky enough to get three spots tonight or in a week. A according to the lady there, it's *impossible*.
```

Comment:
lol, we actually first asked if we can get three seats for tonight. She said no and quickly closed the door. Then we knocked and asked how about in a week. She said impossible then quickly closed the door again 😂